src/portals/hope/features/claims/claim-details/ClaimSubclaims/components/SubclaimType/SubclaimType.tsx

```diff
@@ -27,6 +33,15 @@ gql`
     subclaim(id: $subclaimId) {
       id
       type
+      claimType {
+        id
+        displayName
+        defaultDeductible
+        defaultReserve
+      }
+      contract {
+        availableClaimTypes
```

Comment:
You need the ID for the cache to update properly
```suggestion
      contract {
        id
        availableClaimTypes
```